### ISSUE TITLE
Is there a way to exec a query without handling it as a prepared statement?

### ISSUE THREAD

**Issue by NachoNievaG** [trinodb/trino-go-client#68]
Hi!
First of all, i'm really thankful for this library, the implementation for this library is great.
I've been looking into my cluster overview and noticed that everytime a query is executed from the go client, it's triggered as a prepared statement, so it looks like the following:
`EXECUTE _trino_go USING 'foo' `
i'm currently looking for one of the following:
1. a reason for this to happen
2. a way to avoid this execution pattern.

Again, thank you very much!

**Comment by nineinchnick** [comment]
I think it's done this way to avoid extra logic to check if there are any parameters for the query. Can you describe a use case when this is not desirable? Also, PRs are always welcome.

**Comment by NachoNievaG** [comment]
Hi @nineinchnick, thank you for answering.
So, currently for our scenario, we need to change the `insert_existing_partitions_behavior` value to `OVERWRITE` for some given executions. So for this we have two choices:
1. in the same exec, we trigger 3 queries (one for changing the session var, one as the necessary exec, and one for changing the session var back )
2. create two trino client connections and determine which will trigger which queries. (we have a releation of ~80% needing APPEND and ~20% needing `OVERWRITE`.

But when we came to execute two queries in the same exec fot testing purposes, we got back the following error:
```
trino: query failed (400 Bad Request): "Invalid X-Trino-Prepared-Statement header: line 1:67: mismatched input ';'. Expecting: '%', '*', '+', '-', '.', '/', 'AND', 'AT', 'OR', '[', '||', <EOF>, <predicate>"
```
that's how we found out that every exec was sent as a prepared statement (and also read the trino.go file to confirm), therefore my treasure hunting.

I'll be looking for a way to enhance this behavior and possibly come back with good news :)
